src/silo/query_engine/actions/aggregated.cpp

```diff
@@ -0,0 +1,319 @@
+#include "silo/query_engine/actions/aggregated.h"
+
+#include <chrono>
+#include <functional>
+#include <random>
+#include <utility>
+#include <vector>
+
+#include <tbb/enumerable_thread_specific.h>
+#include <tbb/parallel_for_each.h>
+#include <nlohmann/json.hpp>
+
+#include "silo/database.h"
+#include "silo/query_engine/operator_result.h"
+#include "silo/query_engine/query_parse_exception.h"
+#include "silo/query_engine/query_result.h"
+#include "silo/storage/column/date_column.h"
+#include "silo/storage/column/int_column.h"
+#include "silo/storage/column/string_column.h"
+#include "silo/storage/column_group.h"
+#include "silo/storage/database_partition.h"
+
+namespace silo::query_engine::actions {
+
+size_t getTupleSize(std::vector<config::DatabaseMetadata>& group_by_metadata) {
+   size_t size = 0;
+   for (const auto& metadata : group_by_metadata) {
+      if (metadata.getColumnType() == config::ColumnType::STRING) {
+         size += sizeof(common::String<common::STRING_SIZE>);
+      } else if (metadata.getColumnType() == config::ColumnType::FLOAT) {
+         size += sizeof(double);
+      } else if (metadata.getColumnType() == config::ColumnType::FLOAT) {
+         size += sizeof(int32_t);
```

Comment:
```suggestion
      } else if (metadata.getColumnType() == config::ColumnType::INT) {
         size += sizeof(int32_t);
```
This looks like a copy paste error